### DISCUSSION TITLE
Tally Arbiter to Decimator Quad

### DISCUSSION THREAD

**Discussion by jeffreylsoffer** [josephdadams/TallyArbiter#403] [Q&A]
Hi,

I was looking into have Tally Arbiter feed a Decimator [DMON-QUAD](http://decimator.com/Products/MultiViewers/DMON-QUAD%20MultiViewer/DMON-QUAD.html) to provide tally for a multiviewer.
Right now it seems TSL in Tally Arbiter is network based, so I'm wondering if there's a box that would have Ethernet for networking and a RJ422 over rj45? Or is there a way to do this over USB if plugged into the tally arbiter server?

Here's what the Decimator manual says. This is pinouts of an RJ-45 connector:
<img width="631" alt="CleanShot 2022-03-18 at 16 00 04@2x" src="https://user-images.githubusercontent.com/65000982/159075614-ba67620e-796e-4a68-9fc6-a7bae1fb7861.png">

**Comment by mg-1999** [comment]
If you have a USB relay you could make it work with the relay listener? Just short the right cable to ground and it will light the tally

**Comment by josephdadams** [comment]
You could do something where you send tsl to a pi with a usb to rs485 adapter attached, which listens for the tsl data and then sends it over the serial connection as needed. But obviously that would have to be custom coded. It would keep it as a separate process from the TA server though and make it easier to relocate the quad or the server as needed

**Reply by jeffreylsoffer** [reply]
I see. So there's nothing native built in to tally arbiter that would make this possible. Thanks for answering!

**Reply by mg-1999** [reply]
You can still use the relay listener for tally. You only need to connect pin 1,2,3 or 6 to ground to lightup the tally box. Don't know if that's sufficient

**Reply by jeffreylsoffer** [reply]
Oh got it! I had no idea what you meant by a "relay listener" but i just poked around and found the documents for it! Thanks for your help! https://github.com/josephdadams/TallyArbiter/tree/master/listener_clients/relay-listener

**Comment by allen-harris** [comment]
Hi, 

I mocked this up today using a USB to 485 dongle and some free software from Commfront to create a serial server.

https://www.commfront.com/collections/serial-tcp-ip

Configure the software as a server, set up a tsl client pointing at the ip and port shown in the bottom right, and add actions for each multiviewer window to trigger the tally and label. 

I don't want to run the software forever, so I ordered a ethernet to 485 module on Amazon to do the same thing.

Something like this.

https://www.amazon.ca/RS485-Industrial-Bi-Directional-Transparent-Transmission/dp/B0BJL8YNXP/ref=mp_s_a_1_1?crid=21POWXD39MS78

Now I want to figure out how to get the source names from my videohub to show up on the decimator based on the outputs assigned to the inputs. 

Best Regards. 
Allen

**Reply by tjheideman** [reply]
Hi Allen - Just wondering if you were successful in that endeavour? Did the 485 module work for you?

**Comment by allen-harris** [comment]
Hi,

I did not.  I used the TSL module in Companion to drive an Ethernet 485
converter, which is working.  The 485 converter did work to drive the
Decimator UMD.

Best.
Allen

**Reply by allen-harris** [reply]
I am digging further into this as I try to do some wide area tally with M5Atoms and TA.  I have tried driving my 485 converter from TA, but it has not worked.

**Comment by tjheideman** [comment]
I've successfully been able to use a WaveShare PoE TCP to RS422/485 device and it works with the Decimator. I used the reference documentation from TSL on the serial tech specs for TSL 3.1 and it worked like a charm.

**Reply by tjheideman** [reply]
https://www.amazon.ca/Waveshare-RS232-485-422-POE/dp/B0BN61G4VF
This is the model I'm using; you need to reIP from the factory default, and apply the TSL serial settings but once that's done it's pretty easy.
